scripts/benchmarks/configuration.sh

```diff
@@ -43,4 +43,4 @@ else
 fi
 export EXECUTION="${EXECUTION:-wasm}"
 export ADDITIONAL="${ADDITIONAL:-}"
-export ADDITIONAL_FEATURES="${ADDITIONAL_FEATURES:-}"
\ No newline at end of file
+export ADDITIONAL_FEATURES="${ADDITIONAL_FEATURES:-with-global-disputes}"
```

Comment:
Wow, nice, that you caught that too! So this release build needs `with-global-disputes` feature for battery station.

```suggestion
# TODO(#848) Delete default `with-global-disputes`
export ADDITIONAL_FEATURES="${ADDITIONAL_FEATURES:-with-global-disputes}"
```

Reply:
I'll leave that out to be able to wrap up the release faster, since the comment is not mandatory